R/dots.R

```diff
@@ -57,37 +118,57 @@ dots_list <- function(...,
 }
 #' @rdname dots_list
 #' @export
-#' @examples
-#'
-#' # dots_splice() splices lists marked with splice() as well as bare
-#' # lists:
-#' x <- list(1, 2)
-#' dots_splice(!!! x, 3)
-#' dots_splice(x, 3)
 dots_splice <- function(...,
                         .ignore_empty = c("trailing", "none", "all")) {
   dots <- .Call(rlang_dots_flat_list, environment(), FALSE, .ignore_empty, TRUE)
   names(dots) <- names2(dots)
   dots
 }
 
+#' Splice a list within a vector
```

Comment:
Should we document `dots_splice()` here? And then mark this whole API as questioning